testUpdateHostsFile.py

```diff
@@ -51,10 +46,7 @@ def sep(self):
         return "\\" if sys.platform == "win32" else "/"
```

Comment:
@StevenBlack Why aren't we using https://docs.python.org/2/library/os.html#os.sep there ? :thinking:

Reply:
I dunno.  Can you create a new issue to fix this?